testdata/picard/util/abutting_combined.interval_list

```diff
@@ -0,0 +1,4 @@
+@HD	VN:1.6	SO:coordinate
+@SQ	SN:1	LN:249250621	AS:GRCh37	UR:http://www.broadinstitute.org/ftp/pub/seq/references/Homo_sapiens_assembly19.fasta	M5:1b22b98cdeb4a9304cb5d48026a85128	SP:Homo Sapiens
+1	30000	30003	+	target_1
```

Comment:
if you name the intervals in a meaningful way, you can also have the tests "CONTATENATE=true" and the resulting names here will show which intervals got merged.